README.md

```diff
@@ -181,7 +181,7 @@ Version matrix
 
 | CVCalendar | Swift    | Xcode    | Release Notes |
 | ---------- | -------- | -------- | ------------  | 
-|     1.6    | 4.x      | 8.x, 9.x |     HEAD      |
+|     1.6    | 4.x      | 8.x, 9.x, 10.x |     HEAD      |
```

Comment:
Can you please fix the formatting of the table?

Reply:
Sorry, I dont know what you mean. Isn't it already correctly formatted? Or do you mean to make an extra row for Swift 4.2?

Reply:
Fixed!